libs/dh/wholesale/feature-calculation-steps/src/lib/results/dh-wholesale-results.component.html

```diff
@@ -37,7 +37,7 @@
     ></watt-description-list-item>
     <watt-description-list-item
       [label]="t('calculationPeriod')"
-      [value]="batch?.period | dhDateRange"
+      value="{{ batch?.period?.start | wattDate }} - {{ batch?.period?.end | wattDate }}"
```

Comment:
Question: Maybe we should keep a `DateRange` pipe to make our HTML cleaner? (since our date ranges always have the same format/type)?